src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+// eslint-disable-next-line no-unused-vars
+import { BrowserRouter as Router, Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+export default function Navigation({ onClick }) {
+  return (
+    <NavStyled onClick={onClick}>
+      <LinkStyled to='/'>Dashboard</LinkStyled>
+      <LinkStyled to='/opportunities'>Opportunities</LinkStyled>
+      <LinkStyled to='/my-projects'>My Projects</LinkStyled>
+      <LinkStyled to='/create'>Initiate a Project</LinkStyled>
+    </NavStyled>
+  )
+}
+Navigation.propTypes = {
```

Comment:
Move above line 7 maybe?

Reply:
Done. For some reason, I can't directly answer the previous comment, so I'll do it here: When I omit the Router (which is not being used in the file), the Links are not clickable anymore, no idea why